frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -516,13 +516,17 @@ private void prepareReleaseEdit(RenderRequest request, RenderResponse response)
                 if (isNullOrEmpty(id)) {
                     id = release.getComponentId();
                 }
+                component = client.getComponentById(id, user);
 
             } else {
+                component = client.getComponentById(id, user);
```

Comment:
Maybe do this before the else-block (merge with if-block statement)

Reply:
see above